### ISSUE TITLE
Rule Proposal: Disallow duplicate imports

### ISSUE THREAD

**Issue by SimenB** [eslint/eslint#5453]
If you `import` from the same package multiple time, an error should be issued.

Side note: IntelliJ's `optimize imports` joins duplicate `import`s

Sample:
```js
import { clone } from 'lodash-es';
import { merge } from 'lodash-es';
```

Should be:
```js
import { clone, merge } from 'lodash-es';
```

This is stylistic. And I think it should be in core because it seems like a very general rule, and most people would want to have it turned on.

I would be willing to create this myself, but that might take some time.

**Comment by mysticatea** [comment]
:+1: but this is duplicate of #3478

**Comment by SimenB** [comment]
Ah, thanks! Searching for import gave almost 200 hits, so I didn't find it.
